Doc/whatsnew/3.13.rst

```diff
@@ -227,6 +260,106 @@ Support For Mobile Platforms
   (PEP written and implementation contributed by Russell Keith-Magee in
   :gh:`114099`.)
 
+.. _whatsnew313-jit-compiler:
+
+Experimental JIT Compiler
+=========================
+
+When CPython is configured using the ``--enable-experimental-jit`` option,
+a just-in-time compiler is added which may speed up some Python programs.
+
+The internal architecture is roughly as follows.
+
+* We start with specialized *Tier 1 bytecode*.
+  See :ref:`What's new in 3.11 <whatsnew311-pep659>` for details.
+
+* When the Tier 1 bytecode gets hot enough, it gets translated
+  to a new, purely internal *Tier 2 IR*, a.k.a. micro-ops ("uops").
+
+* The Tier 2 IR uses the same stack-based VM as Tier 1, but the
+  instruction format is better suited to translation to machine code.
+
+* We have several optimization passes for Tier 2 IR, which are applied
+  before it is interpreted or translated to machine code.
+
+* There is a Tier 2 interpreter, but it is mostly intended for debugging
+  the earlier stages of the optimization pipeline.
+  The Tier 2 interpreter can be enabled by configuring Python
+  with ``--enable-experimental-jit=interpreter``.
+
+* When the JIT is enabled, the optimized
+  Tier 2 IR is translated to machine code, which is then executed.
+
+* The machine code translation process uses an architecture called
```

Comment:
```suggestion
* The machine code translation process uses a technique called
```
Feels like a more appropriate term. Happy to stay with "architecture" if that's preferred for some reason though.

Reply:
Fixed.